src/patternfly/components/DataList/styles.scss

```diff
@@ -0,0 +1,199 @@
+  
+  // Item
+  --pf-c-data-list__item--PaddingTop:                     var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--PaddingRight:                   var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--PaddingBottom:                  var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--PaddingLeft:                    var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--MarginLeft:                     var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--BorderColor:                    var(--pf-global--Color--light-200);
+  --pf-c-data-list__item--BorderWidth:                    var(--pf-global--BorderWidth--sm);
+
+  // Toggle
+  --pf-c-data-list__toggle--MarginRight:                  var(--pf-global--spacer--md);
+
+  // offset toggle to align left
+  --pf-c-data-list__toggle--MarginLeft:                   calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-data-list__toggle--Transition:                   var(--pf-global--Transition);
+
+  // Check
+  --pf-c-data-list__check--MarginRight:                   var(--pf-global--spacer--xl);
+
+  // Actions
+  --pf-c-data-list__actions--MarginLeft:                  var(--pf-global--spacer--lg);
+
+  // List item border left
+  --pf-c-data-list__item--before--Color:                  var(--pf-global--active-color--100);
+  --pf-c-data-list__item--before--Width:                  var(--pf-global--BorderWidth--lg);
+  --pf-c-data-list__item--before--Transition:             var(--pf-global--Transition);
+  --pf-c-data-list__item--before--ZIndex:                 var(--pf-global--ZIndex--xl);
+
+  // Expandable content
+  --pf-c-data-list__expandable-content--PaddingTop:       var(--pf-global--spacer--xl);
+  --pf-c-data-list__expandable-content--PaddingBottom:    var(--pf-global--spacer--xl);
+  --pf-c-data-list__expandable-content--MaxHeight:        #{pf-size-prem(600px)};
+  --pf-c-data-list__expandable-content--Transition:       var(--pf-global--Transition);
+  --pf-c-data-list__expandable-content--BorderTopColor:   var(--pf-global--Color--light-200);
+  --pf-c-data-list__expandable-content--BorderTopWidth:   var(--pf-global--BorderWidth--sm);
+
+  // Expanded modifier
+  --pf-c-data-list--m-expanded--BoxShadow:                var(--pf-global--BoxShadow--md);
+  --pf-c-data-list--m-expanded--TransitionDuration:       var(--pf-global--TransitionDuration);
+
+  // spacers are a bit large on mobile view
+  // reduce spacer values for up to 380px
+  @media screen and (max-width: 380px) {
+    --pf-c-data-list__item--PaddingRight:                 var(--pf-global--spacer--lg);
+    --pf-c-data-list__item--PaddingLeft:                  var(--pf-global--spacer--lg);
+    --pf-c-data-list__toggle--MarginRight:                var(--pf-global--spacer--sm);
+    --pf-c-data-list__check--MarginRight:                 var(--pf-global--spacer--md);
+    --pf-c-data-list__actions--MarginLeft:                var(--pf-global--spacer--sm);
```

Comment:
I like this, it's smart and less code than redefining properties, but I'll mention that traditionally we would redefine the properties, so it seems inconsistent. fwiw, @srambach also redefines vars this way in the [progress component](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Progress/progress.scss#L83), and I think we said we were going to discuss writing this way and never did :)